packages/primes/primes.1.0/opam

```diff
@@ -0,0 +1,30 @@
+
+opam-version: "1.2"
+name: "primes"
+version: "1.0"
+maintainer: "Kit Freddura <kitfreddura@gmail.com>"
+authors: "Kit Freddura <kitfreddura@gmail.com>"
+license: "MIT License"
+dev-repo: "https://github.com/KitFreddura/OCaml-Primes.git"
+homepage: "https://github.com/KitFreddura/OCaml-Primes"
+bug-reports: "None yet"
+build: [
+  ["mkdir" "%{prefix}%/lib/primes"]
+]
+ocaml-version: [>= "4.01"]
+install: [
+["ocamlfind" "ocamlc" "-c" "-linkpkg" "-package" "zarith" "primes.mli"]
+["ocamlfind" "ocamlc" "-thread" "-o" "primes.cma" "-linkpkg" "-package" "zarith,core,gen" "primes.ml"]
```

Comment:
These two lines build a bytecode library from `primes.ml`.  You can also build a native code library using 

   [`ocamlfind ocamlopt ...`](http://projects.camlcity.org/projects/dl/findlib-1.2.1/doc/guide-html/c188.html#AEN190).